latte/latte.Config.json

```diff
@@ -0,0 +1,39 @@
+{
+    "LatteDefault": 
+    {
+        "Iterations"     : 5,
+        "Protocol"       : ["tcp", "udp"],
+        "StartPort"      : 50000,
+        "Time"           : 30,
+        "PingIterations" : 10000,
+        "SendMethod"     : [ "b" ],
+        "Default"        : null,
+        "Optimized"      : "-group 0 -rio -riopoll 100000000000",
+        "Options"        : ""
+    },
+
+    "LatteAzure": 
+    {
+        "Iterations"     : 15,
+        "Protocol"       : ["tcp", "udp"],
+        "StartPort"      : 50000,
+        "Time"           : 30,
+        "PingIterations" : 10000,
+        "SendMethod"     : [ "b" ],
+        "Default"        : null,
+        "Optimized"      : "-group 0 -rio -riopoll 100000000000",
+        "Options"        : ""
+    },
+
+    "LatteDetail": {
+        "Iterations"     : 5,

```

Comment:
with -detail flag we want the iterations to be 3 IIRC

Reply:
I had chosen five (5) since PerfViz allows for graphing/visualizing the entire range.  While (3) suffices, I chose (5) for better confidence and negligible increased in runtime.